ui/components/Flex.tsx

```diff
@@ -31,9 +32,11 @@ const Styled = (component) => styled(component)`
 
 class Flex extends React.PureComponent<Props> {
   render() {
-    const { className, children, onMouseEnter, onMouseLeave } = this.props;
+    const { className, children, onMouseEnter, onMouseLeave, testId } =
+      this.props;
```

Comment:
This is forwarding all props, can we do that and give Props the same type as `div`'s Props?

Reply:
I've adjusted it a bit :) , can't pass all props as that includes `tall, wide, wrap` etc which div can't use

Reply:
🎉 .

Ah yeah, I seeeeee.

Hmm, can we collapse the whole thing down?

```tsx

const Flex = styled.div<Props>`
  display: flex;
  flex-direction: ${({ column }) => (column ? "column" : "row")};
  align-items: ${({ align }) => (align ? "center" : "start")};
  ${({ tall }) => tall && `height: 100%`};
  ${({ wide }) => wide && "width: 100%"};
  ${({ wrap }) => wrap && "flex-wrap: wrap"};
  ${({ start }) => start && "justify-content: flex-start"};
  ${({ end }) => end && "justify-content: flex-end"};
  ${({ between }) => between && "justify-content: space-between"};
  ${({ center }) => center && "justify-content: center"};
  ${({ shadow }) => shadow && "box-shadow: 5px 10px 50px 3px #0000001a"};
`;

export default Flex;
```

kind of thing?